### ISSUE TITLE
onnx convert error

### ISSUE THREAD

**Issue by jasonlove521** [dog-qiuqiu/Yolo-Fastest#18]
the code of https://github.com/CaoWGG/TensorRT-YOLOv4 can`t converts the yolo-fastest to onnx model. 
typeerror: buffer is too small for requested array

**Comment by MuhammadAsadJaved** [comment]
@jasonlove521  Have you resolved this issue?  Are you trying to run Nvidia Jetson devices?  Xavier NX etc?

**Comment by jasonlove521** [comment]
@MuhammadAsadJaved  I train the model by using yolov5-torch,  then convert to tensort engine. There is some problem with trt-engine. The mAP and inference speed of trt-engine`s are verlt low.

**Comment by MuhammadAsadJaved** [comment]
Have you tried to convert the Yolo-Fastest model which the provided in the repo?

**Comment by jasonlove521** [comment]
@MuhammadAsadJaved  yes. i have done, but that repo can`t convert the Yolo-Fastest to trt-engine,because Depthwise Convolution　is used in  the Yolo-Fastest, which occur error. i try to modify the code of the provided repo so that it can work, but the performance of trt_engine is not expected with low AP and slow speed.
